### PR TITLE
Win32 docs FAQ

### LINKED ISSUE #15978: [Docs] Windows FAQ Section
*Title*: *Windows FAQ Section*

*Description*:

As discussed in the community meeting we would need to make a FAQ regarding Envoy on Windows.  We should add a Windows Section under the FAQ, https://www.envoyproxy.io/docs/envoy/latest/faq/overview.

Some questions that should answered are:

* What is the minimum version of Windows supported?
* Does Envoy on Windows support FIPS?
* What is the performance of Envoy on Windows?
* Does Envoy on Windows do security releases?
* Which features are not supported on Windows?

Feel free to add more questions.

### PR DESCRIPTION
**Pull request by davinci26**
Commit Message:

Fixes #15978

Additional Description:

As we discussed in the community call we need to have an FAQ section dedicated to Windows. This PR adds also the unsupported features which might get stale as we work. We will address this with #15367

Risk Level: N/A
Testing: N/A
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

### CODE REVIEW

**Comment by davinci26** [comment]
PTAL @phlax and @wrowe 

cc @envoyproxy/windows-dev

**Comment by davinci26** [comment]
@wrowe can you ptal since this is one of the few items pending for the ga

**Comment by phlax** [comment]
docs rendered here https://storage.googleapis.com/envoy-pr/16176/docs/index.html

File: docs/root/faq/windows/win_requirements.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+What are the requirements to run on Envoy on Windows?
+=====================================================
+
+Envoy is tested on Windows Server Core 2019 (Long-Term Servicing Channel). This corresponds to OS version 10.0.17763.1879. We have also tested a few more recent versions of Windows Server
+and in general higher versions will be fully supported. For more info please see `Windows Server Core <https://hub.docker.com/_/microsoft-windows-servercore>`_.
+To build Envoy from source you will need to have at least Windows 10 SDK, version 1803 (10.0.17134.12).
+Earlier versions will not compile because the `afunix.h` header is not available.
```
Comment (phlax) [inline docs/root/faq/windows/win_requirements.rst:7]: use inline literal for `afunix.h` - ie double backtick

File: docs/root/faq/windows/win_requirements.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+There might be Envoy features that require newer versions of Windows. These features will explicitly document the required version.
```
Comment (phlax) [inline docs/root/faq/windows/win_requirements.rst:9]: use inline literal for `afunix.h` - ie double backtick

File: docs/root/faq/windows/win_run_as_service.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
+Can I run Envoy on Windows under SCM?
+=====================================
+
+.. note::
+
+    This feature is still in Experimental state.
+
+
+ You can start Envoy as Windows Service that is managed under `Windows Service Control Manager <https://docs.microsoft.com/en-us/windows/win32/services/using-services/>`_.
```
Comment (phlax) [inline docs/root/faq/windows/win_run_as_service.rst:9]: i think this needs to be dedented - the rest of the page is appearing inside of the note above

File: docs/root/api-v3/config/watchdog/watchdog.rst [hunk 4]
```diff
@@ -1,3 +1,5 @@
+.. _watchdog_overview:
```
Comment (phlax) [inline docs/root/api-v3/config/watchdog/watchdog.rst:1]: this isnt really the overview i think its the api reference

**Review by phlax — CHANGES_REQUESTED** [review]
hi @davinci26 im not qualified to review the correctness of the windows info - but ive left a few api/docs nits

File: docs/root/api-v3/config/watchdog/watchdog.rst [hunk 5]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (wrowe) [inline docs/root/api-v3/config/watchdog/watchdog.rst:2]: Necessary? BOM injection?

Comment (davinci26) [reply docs/root/api-v3/config/watchdog/watchdog.rst:2]: What does BOM stand for?

**Review by wrowe — APPROVED** [review]

**Review by pravb — APPROVED** [review]

File: docs/root/api-v3/config/watchdog/watchdog.rst (hunk 5 above)
Comment (wrowe) [reply docs/root/api-v3/config/watchdog/watchdog.rst:2]: Byte order mark, bit I missed that you had injected the .. topic framing.

**Review by phlax — APPROVED** [review]
lgtm, thanks @davinci26

**Pull request merged**